### DISCUSSION TITLE
Pagination in podcast episode view

### DISCUSSION THREAD

**Discussion by Dragonatorul** [advplyr/audiobookshelf#1772] [Ideas]
I have a podcast that has thousands of episodes and they all load on screen when going to the episode list view. This causes the browser to lag and even stop responding. Limiting the number of episodes returned and loaded to either an infinite scroll, or a paginated view, would prevent this from happening. 

The podcast is https://anchor.fm/s/cff1340/podcast/rss Beau of the Fifth Column, but I imagine it can happen with any podcasts with a lot of episodes. 

Also, not having an easy "select all" option means it's not practical to either mark all episodes as read, or delete them all from the UI. I haven't tried deleting them from the file system because I'm not sure if it would corrupt the database. But this is already open in another discussion.

**Comment by advplyr** [comment]
An issue was created for this #1549 and there is an issue opened in the mobile app repo.

Google podcast manages to load these all on the same page https://podcasts.google.com/feed/aHR0cHM6Ly9hbmNob3IuZm0vcy9jZmYxMzQwL3BvZGNhc3QvcnNz

We can do the same by trimming down things like the tooltips and checkbox toggle. The reason it is so slow is because in Vue we are creating a separate component for each episode that contains a bunch of other components all with event listeners, etc.

We can also do some type of client-side pagination if we need to.
